### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@danroth27  review please

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
closed #1392

**Comment by Rick-Anderson** [comment]
@spboyer please review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,47 +11,38 @@ The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructur
...
-    ```pip install -U sphinx_rtd_theme```
-
-5. Install the Sphinx .NET domain:
-
-    ```pip install sphinxcontrib-dotnetdomain```
-    
-6. Navigate to one of the main project subdirectories in the Docs repo - such as `mvc`, `aspnet`, or `webhooks`.
+    ```git clone https://github.com/aspnet/Docs.git```
+ 
+4. Navigate to the Docs repo and install the build dependencies:
```
Comment (spboyer) [inline CONTRIBUTING.md]: I might suggest putting in the actual commands beneath it helps with new OSS contributors and in this case I had to do a `ls` or `dir` to find out where the `aspnet` folder was.
```
cd aspnet
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,47 +11,38 @@ The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructur
...
 
-7. Run ``make`` (make.bat on Windows, Makefile on Mac/Linux)
+	```pip install -r requirements.txt```
+	
+  **Note**: You might need to run this command from Power Shell.
+	
+5. Navigate to the  *aspnet* directory and run ``make`` (make.bat on Windows, Makefile on Mac/Linux)
```
Comment (spboyer) [inline CONTRIBUTING.md]: I might suggest putting in the actual commands beneath it helps with new OSS contributors and in this case I had to do a `ls` or `dir` to find out where the `aspnet` folder was.
```
cd aspnet
```

**Pull request closed without merging**